.circleci/config.yml

```diff
@@ -2,25 +2,28 @@ version: 2
 jobs:
   build:
     docker:
-      - image: circleci/node:8
+      - image: circleci/node:10
 
     steps:
       - checkout
       - restore_cache:
           name: Restore Yarn Package Cache
           keys:
-            - v1-yarn-packages-{{ checksum "yarn.lock" }}
```

Comment:
Circle CI recommends starting the key with `v1-` so that if we have to change what files are saved, we can change the key to `v2-` etc. to avoid using the old cache